### ISSUE TITLE
Create python package and publish to PyPI

### ISSUE THREAD

**Issue by lisakaser** [nsidc/granule-metgen#62]

**Comment by eigenbeam** [comment]
As an operator, when I want to install metgenc, I can use pip to install it from PyPI.

As a developer, when I want to make a release of metgenc, I can run a command that will trigger a build of the package, and, if successful, publish the new version to PyPI.

As an operator or developer or project manager, when I want to see what was in a new release, I can view the project's CHANGELOG in GitHub.

**Comment by afitzgerrell** [comment]
I'm not sure how best to test this since I already have metgenc installed and I don't know enough to feel confident I can avoid clobbering/wrecking something and getting myself into a pickle. If you have suggestions for how to be brave and go for it, I'll listen and follow.

I took a look at the PyPi site and easily found nsidc-metgenc by searching for "metgenc". There's currently a discrepancy between the GitHub README contents and those on the PyPi nsidc-metgenc page. Is it safe to assume synchronizing those are just on pause during the flurry of development work, and they'll ultimately be unified to match?

**Comment by eigenbeam** [comment]
Since your previous install was via Poetry, this should work:

1. Fire up a new terminal window (and don't run 'poetry shell')
2. Run `pip install --user metgenc`. 
3. This should result in metgenc getting installed from PyPI in your local python user package directory, and when you run `metgenc` it will be this one, not the Poetry-installed version from GitHub

I'm going to update the README to reflect the `--user` option on the pip command.

**Comment by eigenbeam** [comment]
Yeah, the fix for the extra spaces is not yet in a released version, that is, a version that is installable via PyPI.

**Comment by lisakaser** [comment]
👍
